appinventor/appengine/src/com/google/appinventor/client/wizards/youngandroid/NewYoungAndroidProjectWizard.ui.xml

```diff
@@ -1,10 +1,9 @@
-<!-- Copyright 2009-2011 Google, All Rights reserved -->
-<!-- Copyright 2011-2023 MIT, All rights reserved -->
-<!-- Released under the Apache License, Version 2.0 -->
-<!-- http://www.apache.org/licenses/LICENSE-2.0 -->
+<!--NewYoungAndroidProjectWizard.ui.xml -->
```

Comment:
I'm not sure why the copyright information was removed here. Please revert this bit of the change.